.circleci/config.yml

```diff
@@ -6,28 +6,14 @@ workflows:
   php-tests:
     jobs:
       - unit-tests:
-          name: php70
-          version: "7.0"
+          name: php80
+          version: "8.0"
       - unit-tests:
-          name: php71
-          version: "7.1"
-          requires:
-            - php70
+          name: php81
+          version: "8.1"
       - unit-tests:
-          name: php72
-          version: "7.2"
-          requires:
-            - php71
-      - unit-tests:
-          name: php73
-          version: "7.3"
-          requires:
-            - php72
-      - unit-tests:
-          name: php74
-          version: "7.4"
-          requires:
-            - php73
+          name: php82
+          version: "8.2"
```

Comment:
8.3 is getting released in November - it's widely available in RC state in most distros.